### PR TITLE
add $GOLD token

### PR DESCRIPTION
**Pull request by DennisD076**

### CODE REVIEW

File: tokens/main/0xff3bc357600885aaa97506ea6e24fb21aba88fbd/info.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "GOLD Token",
```
Comment (XJWX89) [inline tokens/main/0xff3bc357600885aaa97506ea6e24fb21aba88fbd/info.json:2]: according to the contract, it should be `GOLD Coin`

File: tokens/main/0xff3bc357600885aaa97506ea6e24fb21aba88fbd/additional.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "website": "https://theinkaempirenft.com/",
+    "links": {
+        "twitter": "https://twitter.com/https://twitter.com/INKAEmpireNFT"
```
Comment (XJWX89) [inline tokens/main/0xff3bc357600885aaa97506ea6e24fb21aba88fbd/additional.json:4]: inbalid URL

**Review by XJWX89 — CHANGES_REQUESTED** [review]
hi @DennisD076 , please fix below issues, 
1. Invalid ***twitter URL***
2. ***Token Name***

**Review by DennisD076 — COMMENTED** [review]
Fixed

**Review by XJWX89 — APPROVED** [review]

**Pull request merged**